site/en/docs/extensions/mv3/security/index.md

```diff
@@ -172,8 +170,8 @@ While [content scripts][11] live in an [isolated world][12], they are not immune
   (e.g., [Spectre][14]), and to being taken over by an attacker if a malicious web page compromises
   the renderer process.
 
-Sensitive work should be performed in a dedicated process, such as the extension's [background
-script][15]. Avoid accidentally exposing extension privileges to content scripts:
+Sensitive work should be performed in a dedicated process, such as the extension's service worker. 
+Avoid accidentally exposing extension privileges to content scripts:
```

Comment:
I think this needs more information. It may be more complicated than replacing 'background script' with 'service worker' since the former can do more things than the latter. I don't have a specific suggestion, but I'd start by finding out what 'sensitive work' means.